MiniSim/Service/DeviceParser.swift

```diff
@@ -50,6 +57,58 @@ class IOSSimulatorParser: DeviceParser {
   }
 }
 
+class IOSPhysicalDeviceParser: DeviceParser {
+  struct IOSPhysicalDevicesJson: Codable {
```

Comment:
Let's move this to the `Model` folder SwiftLint is complaining about this being nested.

Reply:
I will but I believe that this struct is an implementation detail of the parsing of ios physical devices and need to stay private to it, perhaps add the private keyword.

Reply:
Yeah that's probably a better idea let's disable this warning

Reply:
done